src/main/java/com/cloudogu/wiki/WikiDispatcherServlet.java

```diff
@@ -53,8 +53,15 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
         String repositoryId = getRepositoryId(req);
         String branchName = getBranchName(req);
 
+
         if (Strings.isNullOrEmpty(repositoryId)) {
-            renderOverview(req, resp);
+            try {
+                renderOverview(req, resp);
+            }
+            catch(RuntimeException ex){
```

Comment:
Do not catch RuntimeException, please catch only the exception which is responsible for the connection problem.